src/Symfony/Component/String/composer.json

```diff
@@ -36,5 +36,8 @@
             "/Tests/"
         ]
     },
+    "conflict": {
```

Comment:
please move the new section before "autoload", after "require-dev"

Reply:
done